docker-test/Dockerfile

```diff
@@ -2,11 +2,10 @@ FROM avenga/docker AS release
 
 # Install bats-core and other requirements
 COPY --from=bats/bats:latest /opt/bats /opt/bats
-COPY install_docker-compose.sh /tmp
+COPY install.sh /tmp
```

Comment:
When you already working on this you could remove the script completely and run the command here with RUN. I think it's always a bad thing to use a script in  Dockerfile, since one can;t see what's happening (e.g. docker history...).